openrtb_ext/imp_automatad.go

```diff
@@ -0,0 +1,6 @@
+package openrtb_ext
+
+type ImpExtAutomatad struct {
```

Comment:
This structure is not needed because there are no usages of it and all bidder parameters are optional.

Reply:
I've updated it and removed the struct. I've still kept the file since it's part of the file checklist on the prebid server documentation.